README.md

```diff
@@ -140,7 +140,7 @@ See also
 [gomplate]: https://docs.gomplate.ca/
 [entr]: https://github.com/eradman/entr
 [testing]: https://www.openpolicyagent.org/docs/latest/policy-testing/
-[hacbsdocs]: https://red-hat-hybrid-application-cloud-build-services-documentation.pages.redhat.com/hacbs-documentation/ec-policies/index.html
```

Comment:
This link is used here:
> Those docs are imported automatically into the official [HACBS Documentation].

Is the new location the official one now? Or do we need to rephrase it?

Reply:
Let's rephrase and point to both.